generator/src/main.rs

```diff
@@ -1650,7 +1650,7 @@ impl Parser {
                     properties.into()
                 }
 
-                pub(crate) fn names(&self) -> Vec<Cow<'static, [u8]>> {
+                pub fn names(&self) -> Vec<Cow<'static, [u8]>> {
```

Comment:
As part of making this public, document what exactly this returns (e.g. does it include nulls?) and when it's useful.